src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -656,8 +656,24 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
 
                 foreach (var action in refactoring.Actions)
                 {
-                    refactoringSuggestedActions.Add(new CodeRefactoringSuggestedAction(
-                        _owner, workspace, _subjectBuffer, refactoring.Provider, action));
+                    if (action.NestedCodeActions.Length > 0)
+                    {
+                        var nestedActions = action.NestedCodeActions.SelectAsArray(
+                            na => new CodeRefactoringSuggestedAction(
+                                _owner, workspace, _subjectBuffer, refactoring.Provider, na));
+
+                        var set = new SuggestedActionSet(categoryName: null,
+                            actions: nestedActions, priority: SuggestedActionSetPriority.Medium, applicableToSpan: applicableSpan);
+
+                        refactoringSuggestedActions.Add(new SuggestedActionWithNestedActions(
+                            _owner, workspace, _subjectBuffer,
+                            refactoring.Provider, action, set));
+                    }
+                    else
+                    {
+                        refactoringSuggestedActions.Add(new CodeRefactoringSuggestedAction(
+                            _owner, workspace, _subjectBuffer, refactoring.Provider, action));
+                    }
```

Comment:
needed to support refactorings with nested options.  Up till now we've never had any of those.